var/spack/repos/builtin/packages/mxnet/package.py

```diff
@@ -110,6 +110,11 @@ def build(self, spec, prefix):
             args.extend(['USE_CUDA_PATH=%s' % spec['cuda'].prefix,
                          'CUDNN_PATH=%s' % spec['cudnn'].prefix,
                          'CUB_INCLUDE=%s' % spec['cub'].prefix.include])
+            # By default, all cuda architectures are built. Restrict only
+            # if a specific list of architectures is specified in cuda_arch.
+            if 'cuda_arch=none' not in spec:
+                cuda_arch = self.spec.variants['cuda_arch'].value
+                args.append('CUDA_ARCH={}'.format(' '.join(self.cuda_flags(cuda_arch))))
```

Comment:
```suggestion
                args.append('CUDA_ARCH={0}'.format(' '.join(self.cuda_flags(cuda_arch))))
```
Needed for Python 2.6 support

Reply:
Indeed, thanks.